packages/core/admin/admin/src/pages/HomePage/ContentBlocks.js

```diff
@@ -19,6 +45,33 @@ const ContentBlocks = () => {
 
   return (
     <Stack spacing={5}>
+      <BlockLink
+        href="https://cloud.strapi.io"
+        target="_blank"
+        rel="noopener noreferrer nofollow"
+        onClick={() => handleClick('didClickOnTryStrapiCloudSection')}
+      >
```

Comment:
I think we can cut quite a few lines of code in this file by mapping on an array of objects instead for all the content boxes